streaming/src/test/scala/org/apache/spark/streaming/CheckpointSuite.scala

```diff
@@ -629,7 +629,7 @@ class CheckpointSuite extends TestSuiteBase with DStreamCheckpointTester
         ssc.graph.getInputStreams().head.asInstanceOf[FileInputDStream[_, _, _]]
       val filenames = fileInputDStream.batchTimeToSelectedFiles.synchronized
          { fileInputDStream.batchTimeToSelectedFiles.values.flatten }
-      filenames.map(_.split(File.separator).last.toInt).toSeq.sorted
+      filenames.map(_.split("/").last.toInt).toSeq.sorted
```

Comment:
This is always "/" because `FileInputDStream.batchTimeToSelectedFiles` is in form of `/a/b/c` via https://github.com/apache/spark/blob/39e2bad6a866d27c3ca594d15e574a1da3ee84cc/streaming/src/main/scala/org/apache/spark/streaming/dstream/FileInputDStream.scala#L207

```scala
scala> import org.apache.hadoop.fs.Path
import org.apache.hadoop.fs.Path

scala> new Path("C:\\a\\b\\c").toString
res0: String = C:/a/b/c
```